### DISCUSSION TITLE
I keep getting automatically declined and I do not know why.

### DISCUSSION THREAD

**Discussion by emillienemann** [community/community#54077] [GitHub Education] [answered]
### Hi everyone,

I've applied 7 times now, on different devices, at different times, at different places, with different proofs, in different browsers.
I keep getting declined immediately each time, and the only reasoning I get is a generic link to the documentation ("Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved"). I've read through all of it, I am elegible, was approved with the same ID in the past, and I included everything. What can I do at this point?

**Comment by emillienemann** [comment, marked as answer]
Small update: I managed to do it on try 10. Switched up the proof type once again and showed a screenshot of my school portal along with my student ID and a paper of my internship. I don't know why this worked, but I'm happy it has.

**Reply by o-zox-o** [reply]
holy fuck why is their system just an rng
